.gitignore

```diff
@@ -1,2 +1,5 @@
 output-*
 modules/optional
+/.vs
+/x86build
+/CMakeSettings.json
```

Comment:
This change seems unrelated. If you think it's necessary, please put it in another PR and explain it there.

Reply:
Was added when using the Visual Studio IDE automatically...